aim/sdk/run.py

```diff
@@ -371,17 +377,28 @@ def _track_impl(
         epoch_view = self.series_run_tree.subtree(metric.selector).array('epoch').allocate()
         time_view = self.series_run_tree.subtree(metric.selector).array('time').allocate()
 
-        max_idx = self.series_counters.get(metric.selector, None)
-        if max_idx is None:
-            max_idx = len(val_view)
-        step = step or max_idx
-
-        self.series_counters[metric.selector] = max_idx + 1
-
-        if max_idx == 0:
+        seq_info = self.sequence_info[metric.selector]
+        if not seq_info.initialized:
+            seq_info.count = len(val_view)
+            seq_info.trace_dtype = self.meta_run_tree.get(('traces', ctx.idx, name, 'dtype'), None)
```

Comment:
`sequence_dtype` instead of `trace_dtype`?

Reply:
👌